Arm/rover_arm_system.hpp

```diff
@@ -43,18 +57,68 @@ class RoverArmSystem : public sjsu::common::RoverSystem
 
   void PrintRoverData(){};
 
-  std::string GETParameters(){};
+  std::string GETParameters(){return "Fill";};
 
-  std::string ParseJSONResponse(){};
+  std::string ParseJSONResponse(){return "Fill";};
 
-  void HandleRoverMovement(){};
+  void HandleArmMovement(){};
 
-  void Homing(){};
+  void HomeArm()
+  {
+    // first getting the accelerometer readings/storing readings
+    Accelerometer::Acceleration_t Rotunda_Acceleration  = Rotunda.GetAccelerometerData();
```

Comment:
```suggestion
    Accelerometer::Acceleration_t rotunda_acceleration  = Rotunda.GetAccelerometerData();
```
bit nitpicky but good to follow the standard